websites/api-docs-smoke-test/package.json

```diff
@@ -10,11 +10,16 @@
     "clean:develop": "yarn clean && yarn develop",
     "build": "npx gatsby build --prefix-paths",
     "prebuild": "yarn clean",
-    "postbuild": "rm -rf ../../public/api-docs-smoke-test && mkdir -p ../../public && mv public ../../public/api-docs-smoke-test"
+    "postbuild": "rm -rf ../../public/api-docs-smoke-test && mkdir -p ../../public && mv public ../../public/api-docs-smoke-test",
+    "generate-ramldoc": "npx commercetools-ramldoc-generator --name test --src ./source-raml/test/api.raml"
   },
   "dependencies": {
     "@commercetools-docs/gatsby-theme-api-docs": "1.8.1",
     "gatsby": "2.19.17",
     "gatsby-cli": "2.8.29"
+  },
+
+  "devDependencies": {
```

Comment:
If that is possible (i.e. resolves the binary) I suggest to move that dependency into the @commercetools-docs/gatsby-theme-api-docs theme package so we don't have to maintain it in every website package.json.